src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -123,13 +123,18 @@ export default defineComponent({
       type: String,
       default: null
     },
+    currentPlaybackRate: {
+      type: Number,
+      default: () => 1
```

Comment:
Numbers are primitives and immutable so they get copied every time, you only need the function when you need to create a new instance of something every time e.g. objects and arrays.
```suggestion
      default: 1
```